cmd/helpers.go

```diff
@@ -18,7 +18,7 @@ import (
 
 const yesNoMsg = "\nContinue [y/N]? "
 
-var nameVerificationMsg = "\nTo continue, enter the name of the %s to be destroyed: "
+var nameVerificationMsg = "\nTo continue, enter the name of the %s to be deregistered: "
```

Comment:
The command name is "unregister" to it's best to use the verb from that "unregistered" instead of "deregistered".